website/layouts/support/list.html

```diff
@@ -13,6 +13,10 @@ <h2><a href="{{ .Permalink }}">Support and Training</a></h2>
                          <a href="https://o11y.eu">
                              <img src="{{ "/o11y.svg" | relURL }}"  alt="" style="width: auto; height: auto; border-color: #ccc;">
                          </a>
+                         <a href="https://tasrieit.com">
+                             <img src="{{ "/tasrie-it-services.png" | relURL }}"  alt="" style="width: auto; height: auto; border-color: #ccc;">
+                         </a>
+                         tasrie-it-services.png
```

Comment:
I think this line is extraneous, shows up on the site as so. Could you kindly resize the image and remove the test @tasrieit?
![Screenshot 2024-01-01 at 6 32 05 PM](https://github.com/thanos-io/thanos/assets/51132453/0a4673b0-92d7-4a1e-96e0-335a47b16afd)

Reply:
i will fix it thank you